Jenkinsfile_CNP

```diff
@@ -16,7 +15,7 @@ properties([
   disableConcurrentBuilds()
 ])
 
-@Library("Infrastructure")
+@Library("Infrastructure@HMAN-514_functionalTest_nonServiceApp")
```

Comment:
Temporary change should be reversed once hmcts/cnp-jenkins-library#987 is released.